tests/ltp/patches/fix_pipe_pipe02.patch

```diff
@@ -0,0 +1,107 @@
+This test case is using two parallel jobs to test the
+“broken pipe” behaviour. sgx-lkl environment
+supports single process environment. Hence,
+test case is modified to work on single process
+environment. write API generating both SIGPIPE
+and EPIPE in musl lib environment. This behaviour
+is also taken care.
```

Comment:
SIGPIPE and EPIPE part is not clear. What is the issue? How did you fix it?

Reply:
done